.gitignore

```diff
@@ -1,7 +1,6 @@
 HELP.md
 target/
-application.properties
-liquebase.properties
+resources/
```

Comment:
should not be excluded from git versioning. If you want to hive sensitive data, just remove it before the commit